test/functional/test_framework/wallet.py

```diff
@@ -182,6 +183,26 @@ def sendrawtransaction(self, *, from_node, tx_hex):
         from_node.sendrawtransaction(tx_hex)
         self.scan_tx(from_node.decoderawtransaction(tx_hex))
 
+    def send_large_transactions(
```

Comment:
since this is being added as a shared helper, would be nice to add a docstring to explain what this function is doing & possibly guidance on how the callers should provide the params.